app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Ingredients</h1>
+
+<% @ingredients.each do |ingredient|  %>
+  <%= "#{ingredient.name}: #{ingredient.cost}" %>
+<% end%>
```

Comment:
Don't forget to include HTML for any rendered erb:

```erb
<% @ingredients.each do |ingredient|  %>
  <p><%= "#{ingredient.name}: #{ingredient.cost}" %></p>
<% end%>
```

In this example, I'm using the `<p>` tag, but you could use a list or something else entirely.